packages/taiko-client/prover/proof_submitter/transaction/sender.go

```diff
@@ -112,12 +112,12 @@ func (s *Sender) Send(
 // latest verified head is not ahead of this block proof.
 func (s *Sender) validateProof(ctx context.Context, proofWithHeader *producer.ProofWithHeader) (bool, error) {
 	// 1. Check if the corresponding L1 block is still in the canonical chain.
-	l1Header, err := s.rpc.L1.HeaderByNumber(ctx, new(big.Int).SetUint64(proofWithHeader.Meta.L1Height+1))
+	l1Header, err := s.rpc.L1.HeaderByNumber(ctx, proofWithHeader.Meta.GetRawBlockHeight())
```

Comment:
Why don't use `+1`?

Reply:
Because of the corresponding protocol change, plz check there.